src/routes/+layout.svelte

```diff
@@ -67,6 +87,7 @@
 		--border-color: rgb(102, 173, 255);
 		--background-color: rgb(12, 44, 76);
 		--text-color: rgb(176, 163, 217);
+		--orange-color: #fa796e;
 		--font-type1: 'Akrobat-black';
 		--font-type2: Arial;
```

Comment:
nit: --main-font and --secondary-font could be a bit cleaner.

Reply:
Done